src/converters/typescript-raw/index.js

```diff
@@ -0,0 +1,168 @@
+const fs = require('fs');
+const rawTokensInterfaceName = 'RawDesignTokens'; // interface name for object format tokens matching the template
+require.extensions['.template'] = function(module, filename) {
+  module.exports = fs.readFileSync(filename, 'utf8');
+};
+const staticInterfaces = require('./interfaces.ts.template');
+
+function convertTokensToTS(tokensByCategory) {
+  return (
+    formatToRawTS(tokensByCategory, rawTokensInterfaceName) +
+    '\n\n' +
```

Comment:
Will these kind of new lines, `\n`, work correctly for both; Unix and Windows?

Reply:
These will not be in the final output, this output goes into tmp ts-files which will be then compiled for js with typings. This was tested with Windows as well, so at least works with node/tsc. Not sure if it would make a difference if some other format was used.

What it practically does is moving the "export" keyword for interfaces to new line.